addons/mail/static/src/components/follower/follower.js

```diff
@@ -51,6 +51,9 @@ export class Follower extends Component {
 }
 
 Object.assign(Follower, {
+    defaultProps: {
+        onClick: () => {},
+    },
```

Comment:
Why bind useless handler? It would better if `undefined` = no handler.

Reply:
In owl 1 undefined handlers are ignored, in owl 2, only empty handlers are allowed (eg `t-on-click.stop=""`), if the handler expression is not empty but is undefined, it throws an error.

Reply:
Thank you for providing the explanation.

It's still not a satisfactory API though and a step back from what we had before, we shouldn't need to define empty functions for something as simple conceptually as deciding when to bind a handler and when not to.